src/lib.rs

```diff
@@ -224,13 +232,23 @@ where
                 archive_reader,
             )?,
             Mode::WriteDisk => {
-                let writer_flags = (ffi::ARCHIVE_EXTRACT_TIME
-                    | ffi::ARCHIVE_EXTRACT_PERM
-                    | ffi::ARCHIVE_EXTRACT_ACL
-                    | ffi::ARCHIVE_EXTRACT_FFLAGS
-                    | ffi::ARCHIVE_EXTRACT_OWNER
-                    | ffi::ARCHIVE_EXTRACT_XATTR) as i32;
-
+                let writer_flags = match options {
+                    Options::PreserveOwnership => {
```

Comment:
Use a `if let` to append the extra flag, so we don't duplicate them.

Reply:
Sure, but what we do with `LossyOwnership` option? This option seems default to me.

Reply:
We can find better name for it. Does `IgnoreOwnership` seems better?

Reply:
Yes, seems better.